packages/manager/modules/storages/src/netapp/netapp.module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'angular-translate';
+
+import component from './netapp.component';
+import share from './share';
+
+import routing from './netapp.routing';
+import service from './netapp.service';
+
+const moduleName = 'ovhManagerNetapp';
+
+angular
+  .module(moduleName, ['ui.router', share])
```

Comment:
[`ovhManagerCore`](https://github.com/ovh/manager/pull/3821/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-6ea1bf869a326ecdf6016067f421b55aR3) is imported (L3) but not injected as a module dependency.

Does this module must be injected?